File: python/0412 FizzBuzz.py
class Solution(object):
    def fizzBuzz(self, n):
        """
        :type n: int
        :rtype: List[str]
        """
        answer = [i for i in range(1,n+1)] 
        for i in range(2,n,3):
            answer[i]="Fizz"
        for i in range(4,n,5):
            answer[i]="Buzz"
        for i in range(14,n,15):
            answer[i]="FizzBuzz"   
            
        return map(str,answer)

File: javascript/0412 Fizz Buzz.js
/**
 * @param {number} n
 * @return {string[]}
 */
var fizzBuzz = function(n) {

    const answer = [];
    for(var i=1;i<=n;i++){

        let string = "";

        if(i%3===0) string+="Fizz";
        if(i%5===0) string+="Buzz";
        if(string==="") string+=i.toString()

        answer.push(string);
    }
    return answer;
};
